src/api/api.js

```diff
@@ -1 +1,21 @@
 export const BASE_URL = 'https://mate-api.herokuapp.com';
+
+export const get = (point, options) => (
```

Comment:
```suggestion
export const get = (endpoint, options) => (
```
name `endpoint` will be more meaningful